scripts/publish/aws-s3.sh

```diff
@@ -31,23 +31,20 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
-  echo "    -k [S3 key prefix]"
   exit 1
 }
 
 ARGV_FILE=""
 ARGV_BUCKET=""
 ARGV_VERSION=""
 ARGV_PRODUCT_NAME=""
-ARGV_PREFIX=""
 
-while getopts ":f:b:v:p:k:" option; do
+while getopts ":f:b:v:p:" option; do
   case $option in
     f) ARGV_FILE="$OPTARG" ;;
     b) ARGV_BUCKET="$OPTARG" ;;
     v) ARGV_VERSION="$OPTARG" ;;
     p) ARGV_PRODUCT_NAME="$OPTARG" ;;
-    k) ARGV_PREFIX="$OPTARG" ;;
```

Comment:
The S3 snapshots repo could still be split into date-specific subfolders? Which would make it _much_ easier to 'manually browse' if we get some kind of UI on top of it.